server/src/app/modules/product/product.controller.ts

```
@@ -45,11 +45,21 @@ const deleteProduct = catchAsync(async (req: Request, res: Response) => {
 });
 
 const findAllProducts = catchAsync(async (req: Request, res: Response) => {
-    const result = await ProductService.findAllProducts();
+    const { page = 1, limit = 10, categoryName, sortBy } = req.query;
+
+    const pageNumber = parseInt(page as string, 10);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)